README.md

```diff
@@ -28,6 +28,12 @@ lc ci
 
 ## Getting Started
 
+### Prerequisites
+
+Please use version Docker 1.11.2 as it is latest supported by elsy.
```

Comment:
I don't think we want to say this, since we support a range of versions. If they already have Docker 1.10, for example, we don't want them to not install `lc`, because they think they can't use it with their version of Docker.